cmd/csi/init/builder.go

```diff
@@ -2,15 +2,19 @@ package init
 
```

Comment:
The goal here was to only add unit tests for the csi-cleanup logic right? Because I ran tests and for example the volume config test fails, because in a previous PR I added the pod namespace to the volumeconfig but apparently forgot to adapt the unit test. Should we do a separate PR to fix the tests before merging to main?